libraries/AP_Math/AP_Math.h

```diff
@@ -18,6 +18,16 @@
 #include "vector2.h"
 #include "vector3.h"
 
+#if defined(__ANDROID__) && defined(__BORLANDC__)
+#include <System.Math.hpp>
```

Comment:
I solve it including into the IDE too :( . I'm thinking in the user, i have included manually these macros about 6. The idea was to do less when an user configure the multiplatform enviroments.